EmergencyStickerEncryption.swift

```diff
@@ -0,0 +1,179 @@
+//
+//  EmergencyStickerEncryption.swift
+//  Krypt
+//
+//  Created by Sun Bin Kim on 21.06.19.
+//
+
+import CryptoSwift
+import Foundation
+
+public class EmergencyStickerEncryption {
+  /// Errors
+  ///
+  /// - invalidSalt: Failed to process salt
+  /// - failToGenerateKeyAndFingerprintFile: Failed to generate key and fingerprint file
+  public enum Error: LocalizedError {
+    case invalidSalt
```

Comment:
Since this is Vivy specific encryption you shouldn't be exposing what the real reason of failure is. 
```suggestion
    case encrypting
    case decrypting
```

These 2 cases should cover all the failures